src/wallet/wallet.cpp

```diff
@@ -421,6 +421,34 @@ void CWallet::SetBestChain(const CBlockLocator& loc)
     SetBestChainINTERNAL(walletdb, loc);
 }
 
+std::set<std::pair<libzcash::PaymentAddress, uint256>> CWallet::GetNullifiersForAddress(const std::set<libzcash::PaymentAddress> & addresses)
+{
+    std::set<std::pair<libzcash::PaymentAddress, uint256>> nullifierSet;
+    for (const auto & txPair : mapWallet)
+        for (const auto & noteDataPair : txPair.second.mapNoteData)
+            if (noteDataPair.second.nullifier && addresses.count(noteDataPair.second.address))
+                nullifierSet.insert(std::make_pair(noteDataPair.second.address, noteDataPair.second.nullifier.get()));
```

Comment:
I generally prefer that completely new code adds braces, but non-blocking because it's relatively clear in these instances. Blocking if another reviewer wants it as well.

Reply:
Yes, please add braces.